docker/data-generator/data-generator.py

```diff
@@ -15,34 +15,49 @@
 from datetime import datetime
 from time import sleep
 
-import redis
+from pymongo import MongoClient
+from pymongo.collection import Collection
 from pytz import timezone
 from sklearn.datasets import load_wine
 
-REDIS_CLINET = redis.Redis(host="redis", port=6379, db=0)
-KR_TZ = timezone("Asia/Seoul")
 
+def create_collection(mongo_client: MongoClient) -> Collection:
+    """Create collection."""
+    database = mongo_client["mongo"]
+    if "wine_data" not in database.list_collection_names():
+        database.create_collection("wine_data")
+    return database["wine_data"]
 
-def main() -> None:
-    """Run main function."""
+
+def generate_data(collection: Collection) -> None:
+    """Generate data."""
     X, y = load_wine(return_X_y=True, as_frame=True)  # noqa: N806
     cnt = 0
     data_length = X.shape[0]
     while True:
-        REDIS_CLINET.set(
-            cnt,
-            json.dumps(
-                {
-                    "Time": datetime.now(timezone("Asia/Seoul")).strftime("%Y-%m-%d %H:%M:%S"),
-                    "X": json.dumps(X.iloc[cnt % data_length].to_dict()),
-                    "y": str(y.iloc[cnt % data_length]),
-                },
-            ),
+        collection.insert_one(
+            {
+                "Time": datetime.now(timezone("Asia/Seoul")).strftime("%Y-%m-%d %H:%M:%S"),
+                "X": json.dumps(X.iloc[cnt % data_length].to_dict()),
+                "y": str(y.iloc[cnt % data_length]),
+            },
```

Comment:
이렇게 하면 몇번째 row인지 알 수 있는 index가 없는 것 같습니다.
다음과 같이 추가하면 어떠신가요?

```suggestion
        collection.insert_one(
            {
                "Time": datetime.now(timezone("Asia/Seoul")).strftime("%Y-%m-%d %H:%M:%S"),
                "index":cnt,
                "X": json.dumps(X.iloc[cnt % data_length].to_dict()),
                "y": str(y.iloc[cnt % data_length]),
            },
```

사실 `Time`을 기준으로 정렬하면 순서는 보장되기 때문에 아주 필요한 작업은 또 아니라고 생각합니다.
기존에 저장되는 구조를 보장하는 것에 대한 의견 부탁드립니다 🙏

Reply:
수정하였습니다~